dev/benchmarking/data_model.dot

```diff
@@ -0,0 +1,219 @@
+/*
```

Comment:
This file is helpful, thanks. Looking at it, I have one question: the project table appears unrelated to any other table? Does it play a role? Is it meant to play one in the future?

Reply:
You are correct. The project is not connected to anything else and currently doesn't play a role.
There is a function "public.project_details()" which will always pull the most recent entry.

The only purpose is for downstream use by a Codespeed / ASV front end, since both link to the project git commits and the project homepage. This means we can either hardcode the project information in the site, or pull it from the database. The project *could* be added to the benchmark, but then it seemed silly because really regardless of the benchmark, we always will want to see the most recent URL for the project and its repository.

One more thing is i moved the "lessisbetter" column from the "benchmark" table to the "benchmark_type" table, even though the recommendation was to move it to the "units" table because I assumed all units of a specific type would strive to either maximize or minimize their metric. (In case that's wrong and I should change it)

Reply:
Sounds fine to me. Thanks for the explanations!